cli/internal/run/real_run.go

```diff
@@ -271,8 +292,29 @@ func (ec *execContext) exec(ctx gocontext.Context, packageTask *nodes.PackageTas
 
 	cmd := exec.Command(ec.packageManager.Command, argsactual...)
 	cmd.Dir = packageTask.Pkg.Dir.ToSystemPath().RestoreAnchor(ec.repoRoot).ToString()
-	envs := fmt.Sprintf("TURBO_HASH=%v", hash)
-	cmd.Env = append(os.Environ(), envs)
+
+	currentState := env.GetEnvMap()
+	passthroughEnv := env.EnvironmentVariableMap{}
+
+	if strictEnv {
+		defaultPassthrough := []string{
```

Comment:
I think Windows will also require `SYSTEMROOT`. Go adds it by default if it's missing, but let's be explicit that we're including it.